packages/cli/src/commands/build-docs/utils.ts

```diff
@@ -36,10 +36,10 @@ export function getObjectOrJSON(
       }
       break;
     default: {
-      if (config) {
```

Comment:
Can we here have a config without the `theme` field? If yes, you should use optional chaining.